src/featureFlags/flags/unifiedRouter.ts

```diff
@@ -0,0 +1,12 @@
+import { BaseVariant, FeatureFlag, useBaseFlag } from '../index'
+
+export function useUnifiedRouterFlag(): BaseVariant {
```

Comment:
minor nit can this be renamed `unifiedRoutingApiFlag` since it still uses the same router but behind a different URL~